support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -107,7 +107,7 @@ header[role="banner"] {
 
   @include mq($from: tablet) {
     height: 115px;
-    padding: 0 0 0 $content-left-margin-tablet;
+    //padding: 0 0 0 $content-left-margin-tablet;
```

Comment:
I would delete this rather than commenting it out

Reply:
true